etc/kayobe/kolla/config/prometheus/prometheus.yml.d/ipmi-exporter.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Same as above, I guess its cool it only needs to drop into this directory, but I guess it was copied here using the above playbook.